exampleSite/content/fragments/_index/list.md

```diff
@@ -5,5 +5,5 @@ weight = 150
 
 count = 1000
 section = "fragments"
-disable_summary = true
+#summary = true
```

Comment:
The default is already true so the comment should probably be false. Also adding a small comment that summary defaults to true, when not set would be good for the user.

Reply:
That one still needs fixing I think.